ament_cmake_test/ament_cmake_test/__init__.py

```diff
@@ -310,7 +316,7 @@ def log(msg, **kwargs):
     return rc
 
 
-def _generate_result(result_file, *, failure_message=None, skip=False, error_message=None):
+def _generate_result(result_file, *, failure_message=None, skip=False, error_message=None, test_time=0):
```

Comment:
The invocation in line 250/255 should probably also pass `test_time`.

Reply:
See 7849d1a